www/assets/js/sloDocument.js

```diff
@@ -222,12 +215,12 @@ const sloDocument = (function() {
         updateURL()
       },
       onEdit : () => {
-        //documentSaveChanges()
+        //saveChangesDocument()
         //hideEditor()
         //updateURL()
       },
-      documentSaveChanges: () => {
-        documentSaveChanges()
+      saveChangesDocument: () => {
+        saveChangesDocument()
         hideEditor()
```

Comment:
Dit ziet eruit als een infiniteloop - het is het niet omdat saveChangesDocument() eerder gedefinieerd staat, maar dan is deze naam een beetje verwarrend.